definitions/ext-sap_mssql/golden_metrics.yml

```diff
@@ -0,0 +1,33 @@
+d0bAvailability:
```

Comment:
```suggestion
dbAvailability:
```

Also since these are already DB's shouldn't this be availability only?

Reply:
Yes. I made the change to took out the "db" in the titles and section label.